src/app/workflow/workflow.js

```diff
@@ -198,6 +202,8 @@ angular.module('app.workflow', [
   };
 
   mediator.subscribeForScope('wfm:workflow:updated', $scope, function(workflow) {
+    //remove generated angular variables
+    workflow = JSON.parse(angular.toJson(workflow));
```

Comment:
Same as above, this topic is coming from the raincatcher-workflow module. It would be better to change it there.

https://github.com/feedhenry-raincatcher/raincatcher-workflow/blob/master/lib/angular/directive.js#L144